src/css/custom.css

```diff
@@ -175,6 +175,10 @@ h2[itemprop='headline'] {
 	--ifm-h2-font-size: 1.75rem;
 }
 
+.markdown > h3 {
```

Comment:
I haven't checked it locally but we may not want this? In #118 I made sure to align everything consistently.

Reply:
**Before: h2 is almost indistinguishable from h3 to my eye:**
![Screenshot from 2023-07-28 13-36-33](https://github.com/transloadit/uppy.io/assets/1199054/23e2c053-c6fd-474a-acdb-74ee872ce02f)

**After:**
![Screenshot from 2023-07-28 13-36-21](https://github.com/transloadit/uppy.io/assets/1199054/c65b052b-714a-4df4-8310-0c4ea16984df)